### PR TITLE
Adds Core Web Vitals tools

### LINKED ISSUE #3050: content: CWV tools

### PR DESCRIPTION
**Pull request by addyosmani**
Fixes #3050 

This adds the Core Web Vitals tooling round-up.

cc @kaycebasques @philipwalton @robdodson

### CODE REVIEW

File: src/site/content/en/blog/vitals-tools/index.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+hero: hero.png
+thumbnail: thumbnail.png
+alt: Chrome User Experience logo, PageSpeed Insights logo, Lighthouse logo, Search Console logo, DevTools logo, Web Vitals extension logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - fast
+---
+
+The recently announced [Web Vitals](/vitals/) initiative provides unified guidance about quality signals that are essential for all sites to deliver a great user experience on the web. We're happy to announce that **all of our popular tools for web developers now support measurement of Core Web Vitals**, helping you more easily diagnose and fix user experience issues. This includes [Lighthouse](https://github.com/GoogleChrome/lighthouse), [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/), [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools), [Search Console](https://search.google.com/search-console/about), the [Web Vitals Chrome extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma) and a new (!) [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report) API. 
+
```
Comment (addyosmani) [inline src/site/content/en/blog/vitals-tools/index.md:19]: HOLD for addition of new paragraph 👋

**Comment by qa8302** [comment]
https://docs.google.com/document/d/1D9PLzj7c1ON-kiSHJ4584W8Pv-Avmfkab2x7esJ8De4/edit

File: src/site/content/en/blog/vitals-tools/index.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+thumbnail: thumbnail.png
+alt: Chrome User Experience logo, PageSpeed Insights logo, Lighthouse logo, Search Console logo, Chrome DevTools logo, Web Vitals extension logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - web-vitals
+  - performance
+---
+
+The recently announced [Web Vitals](/vitals/) initiative provides unified guidance about quality signals that are essential for all sites to deliver a great user experience on the web. We're happy to announce that **all of Google's popular tools for web developers now support measurement of Core Web Vitals**, helping you more easily diagnose and fix user experience issues. This includes [Lighthouse](https://github.com/GoogleChrome/lighthouse), [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/), [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools), [Search Console](https://search.google.com/search-console/about), the [Web Vitals Chrome extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma) and a new (!) [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report) API.
```
Comment (kaycebasques) [inline src/site/content/en/blog/vitals-tools/index.md:19]: FYI I changed `all of our popular tools` to `all of Google's popular tools`

File: src/site/content/en/blog/vitals-tools/index.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+The Web Vitals extension measures the three Core Web Vitals metrics in real-time for (desktop) Google Chrome. This is helpful for catching issues early on during your development workflow and as a diagnostic tool to assess performance of Core Web Vitals as you browse the web. 
+
+The extension is now available to install from the [Chrome Web Store](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)! We hope you find it useful. We welcome any contributions to improve it as well as feedback on the project's [GitHub](https://github.com/GoogleChrome/web-vitals-extension/) repository.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./Vitals-Tools10.png" alt="Core Web Vitals displayed in real-time with the Web Vitals Chrome Extension">
+</figure>
+
+#### Quick Highlights
```
Comment (kaycebasques) [inline src/site/content/en/blog/vitals-tools/index.md:159]:
```suggestion
#### Quick highlights
```

File: src/site/content/en/blog/vitals-tools/index.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
+---
+title: "Tools to measure Core Web Vitals"
+subhead: Your favorite developer tools can now measure the Core Web Vitals metrics.
+authors:
+  - addyosmani
+  - egsweeny
+date: 2020-05-28
+description: |
+  Read about the newly announced Core Web Vitals measurement support across popular web developer tools like Lighthouse, PageSpeed Insights, Chrome UX Report, and many others.
+hero: hero.png
```
Comment (kaycebasques) [inline src/site/content/en/blog/vitals-tools/index.md:10]: I accidentally optimized this but I checked it on staging and it seems fine. If it ends up looking grainy we can just re-upload a higher fidelity version.

P.S. what's up with the 3 straight lines on the DevTools logo 😆 

image

**Comment by kaycebasques** [comment]
@addyosmani you should be able to merge this when it's ready

If you add any new images please optimize them

Please double-check the new text that you add because I probably won't see it before you merge

**Comment by philipwalton** [comment]
I pushed [some commits](https://github.com/GoogleChrome/web.dev/compare/522ed1f...8c95829) to this branch to update all of the metric and Web Vitals guides to point to these new tools (so they'll go live with the tooling update post).

**Comment by addyosmani** [comment]
Thanks, @philipwalton!

File: src/site/content/en/blog/vitals-tools/index.md (hunk 4 above)
Comment (addyosmani) [reply src/site/content/en/blog/vitals-tools/index.md:10]: I'll correct this in my next push. Something funky happened with the SVG import first time around :)

File: src/site/content/en/blog/vitals-tools/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./Vitals-Tools1.png" alt="Summary of Chrome and Search Tools that support the Core Web Vitals metrics">
+</figure>
+
+**_[Quick Refresher]_** Lab tools provide insight into how a _potential user_ will likely experience your website and offer reproducible results for debugging. **Field** tools provide insight into how your _real users_ are experiencing your website; this type of measurement is often called Real User Monitoring (RUM). Each [lab or field tool](/how-to-measure-speed/#lab-data-vs-field-data) offers distinct value for optimizing your user experience. 
+
+To begin your journey optimizing user-experience with Core Web Vitals, try the following workflow:
```
Comment (mihajlija) [inline src/site/content/en/blog/vitals-tools/index.md:27]:
```suggestion
{% Aside 'key-term' %}
Lab tools provide insight into how a _potential user_ will likely experience your website and offer reproducible results for debugging. 
{% endAside %}

{% Aside 'key-term' %}
Field tools provide insight into how your _real users_ are experiencing your website; this type of measurement is often called Real User Monitoring (RUM). 
{% endAside %}

Each [lab or field tool](/how-to-measure-speed/#lab-data-vs-field-data) offers distinct value for optimizing your user experience. To begin your journey optimizing user-experience with Core Web Vitals, try the following workflow: 
```

File: src/site/content/en/blog/vitals-tools/index.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./lighthouse.png" alt="Lighthouse 6.0 showing the latest Core Web Vitals metrics">
+</figure>
+
+Lighthouse 6.0 introduces three new metrics to the report. Two of these new metrics - [Largest Contentful Paint](/lcp/) (LCP) and [Cumulative Layout Shift](/cls/) (CLS) - are lab implementations of Core Web Vitals and provide important diagnostic information for optimizing user experience. Given their importance for assessing user experience, the new metrics are not only measured and included in the report, they are also factored in calculating the performance score.
+
+The third new metric included in Lighthouse - [Total Blocking Time](/tbt/) (TBT) - correlates well with the field metric [First Input Delay](/fid/) (FID), another Core Web Vitals metric. Following the recommendations provided in the Lighthouse report and optimizing against your scores sets you up to provide the best possible experience to your users.
```
Comment (mihajlija) [inline src/site/content/en/blog/vitals-tools/index.md:48]:
```suggestion
Lighthouse 6.0 introduces three new metrics to the report. Two of these new metrics—[Largest Contentful Paint](/lcp/) (LCP) and [Cumulative Layout Shift](/cls/) (CLS)—are lab implementations of Core Web Vitals and provide important diagnostic information for optimizing user experience. Given their importance for assessing user experience, the new metrics are not only measured and included in the report, they are also factored in calculating the performance score.

The third new metric included in Lighthouse—[Total Blocking Time](/tbt/) (TBT)—correlates well with the field metric [First Input Delay](/fid/) (FID), another Core Web Vitals metric. Following the recommendations provided in the Lighthouse report and optimizing against your scores sets you up to provide the best possible experience to your users. 
```

File: src/site/content/en/blog/vitals-tools/index.md (hunk 5 above)
Comment (mihajlija) [reply src/site/content/en/blog/vitals-tools/index.md:27]: "Quick refresher" seems arbitrary, I think key term aside formatting would serve well here :) https://web.dev/handbook/web-dev-components/#key-term-asides

Comment (addyosmani) [reply src/site/content/en/blog/vitals-tools/index.md:27]: Thanks! I'll talk to Elizabeth about us switching over to the key term formatting before we land this :)

**Pull request merged**